docs/configuration-guides/components/metrics-server.md

```diff
@@ -0,0 +1,49 @@
+# Metrics Server Configuration Reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Argument Reference](#argument-reference)
+* [Installation](#installation)
+* [Uninstallation](#uninstallation)
+
+## Introduction
+
+Metrics server is a cluster addon that is required for supporting [Horizontal Pod Autoscaler
+(HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+
+## Prerequisites
+
+* A Lokomotive cluster with `enable_aggregation` set to true.
+
+## Configuration
+
+This component does not require any specific configuration.
+
+Empty configuration block is also accepted as valid configuration.
+
+Metrics server component configuration example:
+
+```hcl
+component "metrics-server" {}
+```
+
+**NOTE**: changing the `enable_aggregation` option once the cluster has already been setup with
```

Comment:
This is not true anymore after we merge #32. Right, @invidian?

Reply:
Yes, with #32 it's possible to enable it after cluster has been created.

Reply:
ok so removing the NOTE, so that the documentation stays updated once #32 gets merged.